templates/accounts/login_signup.html

```diff
@@ -60,6 +60,11 @@ <h1>Welcome Back!</h1>
                     <h1>Hello, Friend!</h1>
                     <p>Enter your personal details and start journey with us</p>
                     <button class="ghost" id="signUp">Sign Up</button>
+                    <div class="social">
+                        <ul>
+                            <button type='submit'><a href="{% url "social:begin" "google-oauth2" %}?next={{ request.path }}">Sign in with Google</a></button>
+                        </ul>
+                    </div>
```

Comment:
The position and style of social auth button doesn't look good here. Instead, put the button in between Sign in and email field with some design like the following:

![Screenshot from 2021-01-27 23-26-48](https://user-images.githubusercontent.com/39924567/106033145-65709e00-60f7-11eb-875b-f325ea66c370.png)